pages/linux/dconf.md

```diff
@@ -0,0 +1,16 @@
+# dconf
+
+> Simple tool for manipulating a dconf database.
```

Comment:
```suggestion
> Simple tool for reading a dconf database.
```

Reply:
It can read, but it can also write and do other things :thinking:

Reply:
So it can, ignore this, I only read the first usage and the filename.
Sorry, new to the job!